SRS/SRS_OC_BODBAW/srs_oc_bodbaw_netcdf_creation.py

```diff
@@ -288,11 +300,14 @@ def create_filename_output(metadata, data):
     date_start = pd.to_datetime(time_values).min().strftime("%Y%m%dT%H%M%SZ")
     date_end   = pd.to_datetime(time_values).max().strftime("%Y%m%dT%H%M%SZ")
 
-    return os.path.join('IMOS_SRS-OC-BODBAW_X_%s_%s-%s_FV02_END-%s_C-%s' % (date_start,
-                                                                       cruise_id,
-                                                                       data_type,
-                                                                       date_end,
-                                                                       date_created))
+    return os.path.join(
+        'IMOS_SRS-OC-BODBAW_X_%s_%s-%s_FV02_END-%s_C-%s' % (date_start,
+                                                            cruise_id,
+                                                            data_type,
+                                                            date_end,
+                                                            date_created)
+    )
```

Comment:
It looks like `os.path.join()` is doing nothing here as it is only given a single string??